docs/migrate.md

```diff
@@ -4,6 +4,19 @@ This document records manual migrations that are necessary to apply when upgradi
 Sourcegraph versions. All manual migrations between the version you are upgrading from and the
 version you are upgrading to should be applied (unless otherwise noted).
 
+## 3.15
+
+### Existing installations: Migrating lsif-server
```

Comment:
```suggestion
### Migrating lsif-server
```

I have no idea why the 3.14 section below says "Existing installations", because this entire document is _only_ about existing installations. Nobody would come here otherwise.